pronto_mia/lib/core/services/graphql_service.dart

```diff
@@ -99,7 +99,8 @@ class GraphQLService {
     }
   }
 
-  bool _isNetworkAvailable(OperationException exception) {
+  // ToDo: Replace with #39 globally.
+  static bool _isNetworkAvailable(OperationException exception) {

```

Comment:
Why did you make this method static?

Reply:
Because it does not need anything from the object. But it can be reverted if you think it should be.